__test__/Block.spec.js

```diff
@@ -3,29 +3,28 @@ import { StyleSheetTestUtils } from 'aphrodite';
 import { configure, mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 configure({ adapter: new Adapter() });
+import { mockUtils as utils, joinClasses } from '@volusion/element-block-utils';
+import { block as Block, defaultConfig } from '../src';
 
-import { block as Block } from '../src/';
-import { defaultConfig } from '../src/configs';
-
+let props;
 describe('The Block', () => {
     StyleSheetTestUtils.suppressStyleInjection();
-
+    beforeEach(() => {
+        props = { utils, joinClasses };
```

Comment:
Can this just be set once instead of in the beforeEach?
```typescript
describe..
   const props = { utils, joinClasses };
```

Reply:
@greg-murray-volusion technically yes, but there are some things that you might want to add to the default props to reset every time in your tests like a product slug or specifying isAmpRequest in the utils. Formatting it like this will just save time if you need that functionality.